content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -0,0 +1,1682 @@
+* The "shadow projection" had to be manually defined. This made it hard and unapproachable to configure shadows to match a given scene.
+
+**Bevy 0.10** adds "cascaded shadow maps", which breaks up the camera's view frustum into a series of configurable "cascades", which each have their own shadow map. This enables shadows in the cascade "close to the camera" to be highly detailed, while allowing shadows "far from the camera" to cover a wider area with less detail. Because it uses the camera's view frustum to define the shadow projections, the shadow quality remains consistent as the camera moves through the scene. This also means that users don't need to manually configure shadow projections anymore. They are automatically calculated!
+
+<video controls loop><source  src="shadow_cascades.mp4" type="video/mp4"/></video>
+
+Notice how the nearby shadows are highly detailed whereas the shadows in the distance become less detailed as they get farther away (which doesn't matter as much because they are far away).
+
+While shadow cascades solve important problems, they also introduce new ones. How many cascades should you use? What is the minimum and maximum distance from the camera where shadows should appear? How much overlap should there be between cascades? These parameters must be dialed in to fit a given scene.
+
+## Environment Map Lighting
+
+<div class="release-feature-authors">authors: @JMS55</div>
+
+Environment maps are a popular and computationally cheap way to significantly improve the quality of a scene's lighting. It uses a cube map texture to provide 360 lighting "from all directions". This is especially apparent for reflective surfaces, but it applies to all lit materials.
+
+This is what the PBR material looks like without environment map lighting:
+
+![env map before](env_map_before.png)
+
+And this is what the PBR material looks like with environment map lighting:
+
+![env map after](env_map_after.png)
+
+For scenes that need constant lighting (especially outdoor scenes), environment maps are a great solution. And because environment maps are arbitrary images, artists have a lot of control over the character of the scene's lighting.
+
+## Depth and Normal Prepass
+
+<div class="release-feature-authors">authors: @icesentry, Rob Swain (@superdump), @robtfm, @JMS55</div>
+
+<video controls loop><source  src="force_field.mp4" type="video/mp4"/></video>
+<p class="release-feature-authors">This effect uses the depth from the prepass to find the intersection between the ground and the force field</p>
+
+Bevy now has the ability to run a depth and/or normal prepass. This means the depth and normal textures will be generated in a render pass that runs before the main pass and can therefore be used during the main pass. This enables various special effects like Screen Space Ambient Occlusion, Temporal Anti Aliasing and many more. These are currently being worked on and should be available in the next release of bevy.
```

Comment:
This section is unusually high on dense jargon. Can we try to briefly explain what a "depth prepass" and "normal prepass" are?

Reply:
That's what I tried doing in the second sentence. I'm not really sure what else I could add. This is generally intended for people already familiar with rendering and won't really impact people that don't want to touch shaders. The images right after also show both textures to try to give a more visual explanation.

Reply:
In other words, the people that know what a depth or normal texture are and want to use it will be happy about it, and for the ones that don't know what it is, I'm not sure this is the best place to teach rendering concepts but the images should help a lot.

Reply:
Yeah I think im ok with the jargon in this context.